src/main/java/org/generationcp/middleware/dao/ims/TransactionDAO.java

```diff
@@ -380,6 +380,9 @@ private void createTransactionRow(final List<TransactionReportRow> transactionRe
 		+ "' WHEN trntype = " + TransactionType.ADJUSTMENT.getId() + "  THEN '" + TransactionType.ADJUSTMENT.getValue()
 		+ "' END) AS transactionType,"//
 		+ "    tr.trnqty AS amount,"//
+		+ "(SELECT SUM(CASE WHEN transaction.trnstat = " + TransactionStatus.CONFIRMED.getIntValue()
```

Comment:
there is any other way to calculate this outside this query? we were avoiding including the calculation of aggregated data for the lots in this query.